src/storefront/liana/Observer/LianaObserver.php

```diff
@@ -82,7 +82,7 @@ public static function commitRedemption($order_id)
 
                 $data = array(
                     'quantity'    => $amount,
-                    'rule'        => strtoupper(get_woocommerce_currency()),
+                    'rule'        => 'rule:liana:discount',
```

Comment:
Why are we updating this? the Pr is about beans_init_data

Reply:
I moved this part on this [PR](https://github.com/trybeans/beans-woocommerce/pull/47)